service/frontend/errors.go

```diff
@@ -117,4 +117,5 @@ var (
 	errBatchOpsMaxWorkflowExecutionCount = serviceerror.NewInvalidArgument("Workflow executions count exceeded.")
 
 	errUpdateWorkflowExecutionAPINotAllowed = serviceerror.NewPermissionDenied("UpdateWorkflowExecution operation is disabled on this namespace.", "")
+	errWorkerVersioningNotAllowed           = serviceerror.NewPermissionDenied("Worker versioning operations is disabled on this namespace.", "")
```

Comment:
```suggestion
	errWorkerVersioningNotAllowed           = serviceerror.NewPermissionDenied("Worker versioning operations are disabled on this namespace.", "")
```

Reply:
Sorry, I already merged this as I thought we'd use a single system-wide flag to control this behavior.
I still believe a single flag is preferable from the operator perspective.
I can fix the grammer in a separate PR.

Reply:
We already had a situation where I was very glad that we had separate controls for batch api and batch workers, for example. I can imagine similar situations with this feature, e.g. if we need to do some schema migration we might want to block updates but allow workflows to continue running. I'd rather do the more flexible thing.

Reply:
Okay, I'll change in a follow up PR